src/pages/signupPage.tsx

```diff
@@ -71,46 +71,46 @@ const SignUp = () => {
   const [error, setError] = useState('')
   const { setAuthStatus } = useAuth()
 
-  const urlParams = new URLSearchParams(window.location.search)
-  const userId = urlParams.get('userId')
-  const secret = urlParams.get('secret')
-  const verifyParams = {
-    userId: userId || '',
-    secret: secret || '',
-  }
```

Comment:
all of these changes highlighted in red color can be safely discarded, as these are the parts of errorneous code causing the problem during signUp process